### PR TITLE
39 backend parent can be able to edit childs info as long as the child is not approved by advisor

### PR DESCRIPTION
**Pull request by iradudemo**

### CODE REVIEW

File: backend/src/services/child.service.ts [hunk 1]
```diff
@@ -65,3 +65,24 @@ export const deleteChild = async (
 };
 
 
+export const editChild = async (
+  childId: string, chilData: RegisterChildDto
+): Promise<IChild> => {
+  if (!childId)
+    throw new HttpException(httpStatus.BAD_REQUEST, `child with id ${childId} not found`);
+
+  const findChild: IChild = await childModel.findByIdAndUpdate(childId, chilData, {
```
Comment (fabrice-niyongabo) [inline backend/src/services/child.service.ts:74]: Have you tested this??
This line should be:
const findChild: IChild = await childModel.findByIdAndUpdate({_id:childId}, chilData, {
    new: true,
    runValidators: true,
  });

Comment (fabrice-niyongabo) [reply backend/src/services/child.service.ts:74]: Child id is _id

I dont know the usage of {
new: true,
runValidators: true}

**Review by fabrice-niyongabo — CHANGES_REQUESTED** [review]
Fix child service

File: backend/src/services/child.service.ts (hunk 1 above)
Comment (iradudemo) [reply backend/src/services/child.service.ts:74]: i was checked this i was working properly

Comment (iradudemo) [reply backend/src/services/child.service.ts:74]: Just because this findByIdAndUpdate i think there is no need of _id:

**Review by fabrice-niyongabo — APPROVED** [review]
Nice

**Pull request merged**